File: jphoto/src/admin/controllers/user/lock/UserLockModel.java
package admin.controllers.user.lock;

public class UserLockModel {

	private int userId;
	private String userName;

	public UserLockModel( final int userId ) {

		this.userId = userId;
	}

	public int getUserId() {
		return userId;
	}

	public String getUserName() {
		return userName;
	}

	public void setUserName( final String userName ) {
		this.userName = userName;
	}
}

File: jphoto/web/common/js/modules/admin/user/lock/user-lock-model.js
define( ["backbone"], function ( Backbone ) {

	var UserLockModel = Backbone.Model.extend( {

		initialize:function ( options ) {
			this.url = options.baseUrl + "/json/admin/members/" + options.userId + "/lock/";
		}

	 } );

	return {UserLockModel:UserLockModel};
} );
